pkg/grpc/default_api/api_subscribe.go

```
@@ -0,0 +1,267 @@
+}
+
+func (a *api) publishMessageForStream(ctx context.Context, msg *pubsub.NewMessage, pubsubName string) error {
+	a.streamer.lock.RLock()
+	key := a.streamer.StreamerKey(pubsubName, msg.Topic)
+	conn, ok := a.streamer.subscribers[key]
+	a.streamer.lock.RUnlock()
+	if !ok {
+		return fmt.Errorf("no streamer subscribed to pubsub %q topic %q", pubsubName, msg.Topic)
+	}
+
+	envelope, cloudEvent, err := a.envelopeFromSubscriptionMessage(ctx, msg)
```

golangci-lint

ineffectual assignment to err

(ineffassign)